content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md

```diff
@@ -124,10 +124,79 @@ You can also create a Markdown hyperlink by highlighting the text and using the
 
 {% data reusables.repositories.section-links %}
 
+If you need to determine the anchor for a heading in a file you are editing, you can use the following basic rules:
+
+* Letters are converted to lower-case.
+* Spaces are replaced by hyphens (`-`). Any other whitespace characters are removed.
```

Comment:
This now misses any info about punctuation. (Which is perhaps better, as the rules are somewhat wild…)

I think `.;+()` is removed, but the last time i checked `?` not etc. — is there a rule for these characters?

Reply:
```suggestion
* Spaces are replaced by hyphens (`-`). Any other whitespace or punctuation characters are removed.
```